trustlessness.adoc

```diff
@@ -16,34 +16,34 @@ trustless". What do they mean by trustless? Pieter Wuille explains
 this widely used term on
 https://bitcoin.stackexchange.com/a/45674/69518[Stack Exchange]:
 
-[quote, Pieter Wuille, Bitcoin Stack Exchange]
+[quote, Pieter Wuille, Bitcoin Stack Exchange (2016)]
 ____
 The trust we're talking about in "trustless" is an abstract technical
 term. A distributed system is called trustless when it does not
 require any trusted parties to function correctly.
 ____
 
 In short, the word _trustless_ refers to a property of the Bitcoin
-protocol that means it can logically function without "any trusted
-parties". This is different from the trust that you inevitably have to
-put into the software or hardware you run. More on this trust further
-down this chapter.
-
-In centralized systems, we rely on a central actor's reputation
-that they've taken care of security, or roll back in case of issues,
-and on the legal system to enforce violations. These trust
-requirements are problematic in pseudonymous decentralized systems. We
-can't have recourse so we really can't have trust. In
-https://bitcoin.org/bitcoin.pdf[the Bitcoin paper], Satoshi Nakamoto
-describes this problem in his introduction:
-
-[quote,Satoshi Nakamoto,The Bitcoin paper]
+protocol whereby it can logically function without "any trusted
+parties". This is different from the trust you inevitably have to
+put into the software or hardware you run. More on this latter aspect of trust will be discussed further
+in this chapter.
+
+In centralized systems, we rely on a central actor's reputation in order to make sure
+that they will take care of security or roll back in case of issues,
+as well as on the legal system to sanction any violations. These trust
+requirements are problematic in pseudonymous decentralized systems - there is
+no possibility of recourse so there really can't be any trust. In the introduction to
+https://bitcoin.org/bitcoin.pdf[the Bitcoin whitepaper], Satoshi Nakamoto
+describes this problem:
+
+[quote,Satoshi Nakamoto, Bitcoin: A Peer-to-Peer Electronic Cash System (2008)]
 ____
 Commerce on the Internet has come to rely almost exclusively on
 financial institutions serving as trusted third parties to process
 electronic payments. While the system works well enough for most
 transactions, it still suffers from the inherent weaknesses of the
-trust based model.  Completely non-reversible transactions are not
+trust based model. Completely non-reversible transactions are not
```

Comment:
I prefer to keep quotes exactly as is. The double space after period is Nakamoto's way of writing.